planetiler-core/src/main/java/com/onthegomap/planetiler/util/Hashing.java

```diff
@@ -0,0 +1,23 @@
+package com.onthegomap.planetiler.util;
+
+public final class Hashing {
```

Comment:
Can you add a short javadoc to this class and the public members that explain what it's doing/how to use?